topics/testing.md

```diff
@@ -0,0 +1,15 @@
+# Testing
+
+## Books
+
+### [Python Testing with pytest] (https://www.amazon.com/Python-Testing-pytest-Effective-Scalable/dp/1680502409/)
```

Comment:
The link formatting doesn't work with the space between the `] (`, also probably best to follow the samples of other resources of using a bulleted list instead of all headings:

```
## Books

* [Python Testing with pytest](https://...)
```